selenium/package.json

```diff
@@ -6,9 +6,9 @@
   "repository": "dequelabs/axe-core-maven-html",
   "scripts": {},
   "dependencies": {
-    "axe-core": "^4.4.2"
+    "axe-core": "^4.4.3"
   },
   "devDependencies": {
-    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures"
+    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures#v1"
```

Comment:
```suggestion
    "axe-test-fixtures": "github:dequelabs/axe-test-fixtures"
```

Reply:
So the base branch in this repo is v1 so this should be correct https://github.com/dequelabs/axe-test-fixtures/